lib/handler/fastcgi.c

```diff
@@ -548,6 +548,10 @@ static int fill_headers(h2o_req_t *req, struct phr_header *headers, size_t num_h
         }
     }
 
+    /* add date: if it's missing from the response */
+    if (h2o_find_header(&req->res.headers, H2O_TOKEN_DATE, 0) < 0)
```

Comment:
Could we use `!= -1` as a check? It means the same, but it might be a good idea to be consistent with other parts of the code that invoke the same function.

Reply:
done in 79e5186